analyticsdataserver/tests/test_clients.py

```diff
@@ -2,20 +2,31 @@
 import unittest.mock as mock
 
 import responses
+from django.conf import settings
 from django.test import TestCase
 from requests.exceptions import ConnectionError as RequestsConnectionError
 
 from analyticsdataserver.clients import CourseBlocksApiClient
 
 
 class ClientTests(TestCase):
-    @mock.patch('analyticsdataserver.clients.EdxRestApiClient')
-    def setUp(self, restApiClientMock):
-        self.client = CourseBlocksApiClient('http://example.com/', 'token', 5)
+
+    def setUp(self):
+        self.client = CourseBlocksApiClient(settings.LMS_BASE_URL, 'client_id', 'client_secret')
 
     @responses.activate
     def test_all_videos(self):
-        responses.add(responses.GET, 'http://example.com/blocks/', body=json.dumps({'blocks': {
+        responses.add(
+            responses.POST,
+            f'{settings.LMS_BASE_URL}/oauth2/access_token',
+            status=200,
+            body=json.dumps({
```

Comment:
It seems we could use `json` param instead of `body` and omit encoding of the dict.

Reply:
updated